### PR TITLE
add keyboard shortcuts

### PR DESCRIPTION
**Pull request by oscfdezdz**
This is done on top of https://github.com/daitj/gnome-display-brightness-ddcutil/pull/34, once it is merged, I will rebase this one.

imagen

Fixes https://github.com/daitj/gnome-display-brightness-ddcutil/issues/21

### CODE REVIEW

File: display-brightness-ddcutil@themightydeity.github.com/extension.js [hunk 1]
```diff
@@ -342,3 +341,35 @@ function openPrefs() {
...
     }
 }
+
+function increase() {
+    GLib.spawn_command_line_async(`${ddcutil_path} setvcp 10 + 10 --bus 3`)
+}
+
+function decrease() {
+    GLib.spawn_command_line_async(`${ddcutil_path} setvcp 10 - 10 --bus 3`)
+}
```
Comment (oscfdezdz) [inline display-brightness-ddcutil@themightydeity.github.com/extension.js:351]: One of the problems are these functions that are probably replaceable by `setBrightness` if modified a bit or some of the functions already implemented, as you can see it needs the display parameter for `--bus`. It is set to 3 for my setup.

**Comment by oscfdezdz** [comment]
The main issue is that it lacks support for multiple monitors.

File: display-brightness-ddcutil@themightydeity.github.com/extension.js (hunk 1 above)
Comment (oscfdezdz) [reply display-brightness-ddcutil@themightydeity.github.com/extension.js:351]: In addition, quickly spawning the shortcut does not change the brightness and reaching the minimum or maximum brightness gives many errors.

**Comment by oscfdezdz** [comment]
Slider also needs to be updated when using the keyboard shortcuts.

File: display-brightness-ddcutil@themightydeity.github.com/extension.js (hunk 1 above)
Comment (daitj) [reply display-brightness-ddcutil@themightydeity.github.com/extension.js:351]: You could change this like this: increase and decrease funciton would call event emitting function with event type scroll up and scroll down event to the mainMenuButton, what you have implemented in #35 

Multiple monitors will also be handled by that and slider will also update their value accordingly

**Comment by daitj** [comment]
Main comments are [here](https://github.com/daitj/gnome-display-brightness-ddcutil/pull/36#discussion_r777147572)

But I will add that myself and update the project. 

Thanks for you contribution in this project. Once everything looks good on my end, I will merge the project and publish new version in gnome extensions repo.

**Pull request merged**